db/migrate/20160730022819_remove_default_username_from_users.rb

```diff
@@ -0,0 +1,26 @@
+class RemoveDefaultUsernameFromUsers < ActiveRecord::Migration
+  # User with nil username will receive their emails as username:
+  # eg:
+  #   user.test@gmail.com => user.test
+  #   user.test@live.com  => user.test.live
+  def up
+    User.where(username: '').each do |user|
```

Comment:
I thought this too, but this code was already up on own servers, I added it here just to play nice with our dev databases. :)
Can I merge it?